src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -376,7 +391,7 @@ public boolean requiresReauthentication() {
 
   /** Get the access token for the user credentials. */
   public AccessToken getUserAccessToken() {
-    return GoogleOauth.getAccessToken(userCredentials);
+    return GoogleOauth.getAccessToken(googleCredentials);
```

Comment:
I think this method will [try to refresh](https://github.com/DataBiosphere/terra-cli/blob/f27ee455a91fa2f6b1eb60cfc4682d6447af3729/src/main/java/bio/terra/cli/service/GoogleOauth.java#L257) the credentials, but only log (not throw) an error if the refresh fails. Does this refresh work for app-default creds? I'm not sure if you can just call `googleCredentials.refreshIfExpired()` on the credential like for end-user credentials, or whether you need to call `GoogleCredentials.getApplicationDefault()` again.